<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>511</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>pet console</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="console">
         <attribute name="title">
          <string>Console</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Model setting</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QRadioButton" name="showModelBtn">
                 <property name="text">
                  <string>show model</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                 <property name="autoExclusive">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="freeMoveBtn">
                 <property name="text">
                  <string>model freemove</string>
                 </property>
                 <property name="autoExclusive">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="soundsEffectBtn">
                 <property name="text">
                  <string>sounds effect</string>
                 </property>
                 <property name="autoExclusive">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QLabel" name="walkingSpeedLable">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>walking speed: 2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="walkingSpeedSlider">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximum">
                 <number>3</number>
                </property>
                <property name="value">
                 <number>2</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <widget class="QLabel" name="soundValueLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>sound value: 50</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="soundValueSlider">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="value">
                   <number>50</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Maximum</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>25</width>
                  <height>195</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="Line" name="line_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QPushButton" name="resourceFileBtn">
                  <property name="text">
                   <string>open resource file</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="resetBtn">
                  <property name="text">
                   <string>reset</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="schedule">
         <attribute name="title">
          <string>Schedule</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <property name="topMargin">
           <number>29</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="spacing">
             <number>4</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>14</number>
            </property>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Schedule setting</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_4" rowstretch="5,0" columnstretch="0,0,0">
               <property name="sizeConstraint">
                <enum>QLayout::SetNoConstraint</enum>
               </property>
               <property name="leftMargin">
                <number>6</number>
               </property>
               <property name="bottomMargin">
                <number>6</number>
               </property>
               <property name="horizontalSpacing">
                <number>9</number>
               </property>
               <property name="verticalSpacing">
                <number>10</number>
               </property>
               <item row="0" column="0">
                <widget class="QDateEdit" name="dateEdit"/>
               </item>
               <item row="0" column="2">
                <widget class="QLineEdit" name="ScheduleEdit">
                 <property name="placeholderText">
                  <string>enter your schedule</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QTimeEdit" name="timeEdit"/>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="addScheduleBtn">
                 <property name="text">
                  <string>add schedule</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1" colspan="2">
                <widget class="QLabel" name="stateLable">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButton">
              <property name="text">
               <string>schedule reminding</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="ScheduleListView">
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectItems</enum>
            </property>
            <property name="textElideMode">
             <enum>Qt::ElideNone</enum>
            </property>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>220</number>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderDefaultSectionSize">
             <number>30</number>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>300</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="clearFinishTaskBtn">
              <property name="text">
               <string>clear finish schedule</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>682</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
